apps/design/backend/src/worker/generate_election_package_and_ballots.ts

```diff
@@ -181,7 +186,13 @@ export async function generateElectionPackageAndBallots(
     JSON.stringify(systemSettings, null, 2)
   );
 
+  const isCloudTranslationAndSpeechSynthesisEnabled =
+    isVxOrSliOrg(orgId) &&
```

Comment:
One small gotcha here is that this org ID appears to be the user org ID rather than the election org ID. So this means that when we, as Vx, go to export an election package for an NH org, we will end up generating audio.

I think there's a simple fix here, to use the election org ID when generating the background job rather than the user org ID. I'll make that tweak. Not a big deal in the meantime since we're done with NH exports (knock on wood) for this ballot production run.

Reply:
https://github.com/votingworks/vxsuite/pull/6041